include/zephyr/acpi/acpi.h

```diff
@@ -40,6 +58,55 @@ struct acpi_mcfg {
 	ACPI_MCFG_ALLOCATION pci_segs[];
 } __packed;
 
+struct acpi_irq_resource {
+	uint32_t flags;
+	union {
+		uint16_t irq;
+		uint16_t irqs[CONFIG_ACPI_IRQ_VECTOR_MAX];
+	};
+	uint8_t irq_vector_max;
+};
+
+struct acpi_reg_base {
+	enum acpi_res_type type;
+	union {
+		uintptr_t mmio;
+		uintptr_t port;
+	};
+	uint32_t length;
+};
+
+struct acpi_mmio_resource {
+	struct acpi_reg_base reg_base[CONFIG_ACPI_MMIO_ENTRIES_MAX];
+	uint8_t mmio_max;
+};
+
+/**
+ * @brief Get the ACPI HID for a node
+ *
+ * @param node_id DTS node identifier
+ * @return The HID of the ACPI node
+ */
+#define ACPI_DT_HID(node_id) DT_PROP_OR(node_id, acpi_hw_id, "INVALID")
+
+/**
+ * @brief Get the ACPI HID for a node
+ *
+ * @param node_id DTS node identifier
+ * @return The HID of the ACPI node
+ */
+#define ACPI_DT_UID(node_id) DT_PROP_OR(node_id, acpi_uid, "INVALID")
```

Comment:
Looks like the documentation is wrong (probably copy-paste mistake from `ACPI_DT_HID()`. Also as previously noted, please don't use `DT_PROP_OR()` rather deal with he lack of the property in the code itself using `DT_NODE_HAS_PROP()` or similar (possibly combined with an assert)